packages/playwright-ct-angular/registerSource.mjs

```diff
@@ -0,0 +1,137 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// @ts-check
+// This file is injected into the registry as text, no dependencies are allowed.
+
+/** @typedef {import('../playwright-ct-core/types/component').ObjectComponent} ObjectComponent */
+
+import 'zone.js';
+import { reflectComponentType } from '@angular/core';
+import { getTestBed, TestBed } from '@angular/core/testing';
+import {
+  BrowserDynamicTestingModule,
+  platformBrowserDynamicTesting,
+} from '@angular/platform-browser-dynamic/testing';
+
+/** @type {WeakMap<import('@angular/core/testing').ComponentFixture, Record<string, import('rxjs').Subscription>>} */
+const __pwOutputSubscriptionRegistry = new WeakMap();
```

Comment:
Instead of a `WeakMap`, we usually use a symbol. However, in this case it seems like we can combine this map with `__pwFixtureRegistry`?

```
// rootElement -> fixture, subscriptions
Map<Element, {
  fixture: import('@angular/core/testing').ComponentFixture,
  subscriptions: Record<string,  import('rxjs').Subscription>
}>
```